openedx_ledger/models.py

```diff
@@ -128,6 +132,7 @@ class Meta:
         blank=True,
         null=True,
     )
+    history = HistoricalRecords()
```

Comment:
We don't need `history` here because the two inheriting models have history of their own.